types/stripe/stripe-tests.ts

```diff
@@ -765,6 +765,9 @@ stripe.accounts.retrieve(
     "acct_17wV8KBoqMA9o2xk").then(
     (account) => {
         // asynchronously called
+
+        // account should have external_accounts property
+        const externalAccounts: Stripe.IList<Stripe.cards.ICard | Stripe.bankAccounts.IBankAccount> = account.external_accounts;
```

Comment:
nit: prefer:
```
account.external_accounts // $ExpectType IList<ICard> | undefined
```

Reply:
I tried to use `$ExpectType` following your suggestion but I get a linter error. It seems like it's not considering the optionality of `external_accounts`. I'm not too familiar with how to use this. Can you help me?

```
Error: /Users/ethan/work/DefinitelyTyped/types/stripe/stripe-tests.ts:770:9
ERROR: 770:9  expect  TypeScript@next expected type to be:
  IList<ICard | IBankAccount> | undefined
got:
  IList<ICard | IBankAccount>
```

Reply:
Ah, I guess that's fine then. Just set it to `// $ExpectType IList<ICard | IBankAccount>`.

Reply:
I will open an issue with [microsoft/dtslint](https://github.com/microsoft/dtslint/) as I think this is a bug

Reply:
https://github.com/microsoft/dtslint/issues/238